mods/default/tools.lua

```diff
@@ -4,9 +4,7 @@
 local S = default.get_translator
 
 -- The hand
-minetest.register_item(":", {
-	type = "none",
-	wield_image = "wieldhand.png",
+minetest.override_item("", {
```

Comment:
This is ok but please add a comment referring to where the original item is registered.
This also allows seeing what the other itemdef fields are.
I suggest:
`-- The hand`
`-- Override the hand item registered in the engine in builtin/game/register.lua`

Reply:
Done.

(Though, imo the hand's definition should be documented in `lua_api.txt` as overriding it is very common for subgames. Modders shouldn't be required to RTFS.)